packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Video/video.css

```diff
@@ -0,0 +1,28 @@
+.root {
+    font-size: 0;
+}
+
+.inner {
+    box-sizing: border-box;
```

Comment:
Likewise, I believe this inherits from our `index.css`.

Reply:
All instances have been removed 😄 https://github.com/magento/pwa-studio/pull/1872/commits/dc8fb48bc7c2edba5bd3573da6e9fa28e48adfc1